package/features/viz.feature

```diff
@@ -28,17 +28,30 @@
 
 
 Feature: Viz plugin in new project
+    Background:
+        Given I have prepared a config file with example code
+
+    Scenario: Execute viz with Kedro 0.14.0
+        Given I have installed kedro version "0.14.0"
+        And I have run a non-interactive kedro new
+        When I execute the kedro viz command "viz"
+        Then kedro-viz should start successfully
+
+    # `load_context` function was in Kedro 0.14.3, but not KedroContext class
```

Comment:
Sorry it wan't clear enough. I meant why I am adding a test for 0.14.3 and not 0.14.1 nor 0.14.2 (because of `load_context` was introduced in 0.14.3).